src/components/PostPageDetail/index.js

```diff
@@ -614,28 +624,38 @@ const PostPageDetailIdComponent = (props) => {
                 />
               </View>
             </ScrollView>
-            {comments.length > 0 && (
-              <ContainerComment
-                feedId={feedId}
-                itemParent={item}
-                comments={comments}
-                isLoading={loadingPost}
-                refreshComment={handleRefreshComment}
-                refreshChildComment={handleRefreshChildComment}
-                navigateToReplyView={(data) =>
-                  navigateToReplyView(
-                    data,
-                    updateParentPost,
-                    findCommentAndUpdate,
-                    item,
-                    updateVoteLatestChildren,
-                    getComment
-                  )
-                }
-                findCommentAndUpdate={findCommentAndUpdate}
-                contextSource={contextSource}
-                updateVote={handleUpdateVote}
-              />
+            {loadingGetComment ? (
+              <View>
+                {Array.from(Array(10).keys()).map((_, index) => (
+                  <View key={index} style={{marginVertical: normalize(10)}}>
+                    <Shimmer height={20} width={displayWidth} />
+                  </View>
+                ))}
+              </View>
+            ) : (
+              comments.length > 0 && (
+                <ContainerComment
+                  feedId={feedId}
+                  itemParent={item}
+                  comments={comments}
+                  isLoading={loadingPost}
+                  refreshComment={handleRefreshComment}
+                  refreshChildComment={handleRefreshChildComment}
+                  navigateToReplyView={(data) =>
+                    navigateToReplyView(
+                      data,
+                      updateParentPost,
+                      findCommentAndUpdate,
+                      item,
+                      updateVoteLatestChildren,
+                      getComment
+                    )
+                  }
+                  findCommentAndUpdate={findCommentAndUpdate}
+                  contextSource={contextSource}
+                  updateVote={handleUpdateVote}
+                />
+              )
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering based on the `loadingGetComment` state is a good approach to enhance user experience by showing a loading state. However, ensure that `loadingGetComment` is set to false once the comments are fetched successfully or in case of an error.
```diff
- {comments.length > 0 && (
+ {loadingGetComment ? (
+   <View>
+     {Array.from(Array(10).keys()).map((_, index) => (
+       <View key={index} style={{marginVertical: normalize(10)}}>
+         <Shimmer height={20} width={displayWidth} />
+       </View>
+     ))}
+   </View>
+ ) : (
+   comments.length > 0 && (
      <ContainerComment
        feedId={feedId}
        itemParent={item}
        comments={comments}
        isLoading={loadingPost}
        refreshComment={handleRefreshComment}
        refreshChildComment={handleRefreshChildComment}
        navigateToReplyView={(data) =>
          navigateToReplyView(
            data,
            updateParentPost,
            findCommentAndUpdate,
            item,
            updateVoteLatestChildren,
            getComment
          )
        }
        findCommentAndUpdate={findCommentAndUpdate}
        contextSource={contextSource}
        updateVote={handleUpdateVote}
      />
    )
)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->